src/modules/share/share.repository.ts

```diff
@@ -276,7 +272,7 @@ export class SequelizeShareRepository implements ShareRepository {
           required: false,
         },
         {
-          model: this.folderModel,
+          model: FolderModel,
```

Comment:
Same with this

Reply:
It was like a solution for the circular dependency that it showed me when adding the relations between folder and share and private share since the injected model was only being used for relations, I will rollback this